state/containerDB.go

```diff
@@ -151,6 +157,14 @@ func (css *containerStateService) ContainerRegister(c domain.ContainerIface) err
 		)
 	}
 
+	// A per-container mountInfoParser object will be created here to hold the
+	// mount-state created by sysbox-runc during container initialization.
+	mip, err := css.mts.NewMountInfoParser(currCntr, c.InitPid(), true)
```

Comment:
Question: do we need "deep" parsing here? Or could we do with the shallower parsing (to make the container registration faster)?